modules/fundamental/src/advisory/endpoints/test.rs

```diff
@@ -416,9 +417,9 @@ async fn upload_default_csaf_format(ctx: TrustifyContext) -> Result<(), anyhow::
         .to_request();
 
     let response = actix_web::test::call_service(&app, request).await;
-    let doc: Value = actix_web::test::read_body_json(response).await;
-    log::debug!("{doc}");
-    assert_eq!(doc.as_str(), Some("CVE-2023-33201"));
+    let id: Id = actix_web::test::read_body_json(response).await;
+    log::debug!("{id}");
+    assert!(matches!(id, Id::Uuid(_)));
```

Comment:
As a user, I took some comfort seeing the identifier of the doc I posted returned upon success. The opaque uuid gives me less comfort. Not for this PR, but do you think it's worth returning a struct with `uuid` and `identifier` fields upon success?